server/athenian/api/controllers/invitation_controller.py

```diff
@@ -106,15 +106,8 @@ def bad_req():
         is_admin = acc_id == admin_backdoor
         if is_admin:
             # create a new account for the admin user
-            acc_id = await conn.execute(
-                insert(Account).values(Account().create_defaults().explode()))
-            if acc_id >= admin_backdoor:
-                await conn.execute(delete(Account).where(Account.id == acc_id))
-                return ResponseError(GenericError(
-                    type="/errors/LockedError",
-                    title=HTTPStatus.LOCKED.phrase,
-                    status=HTTPStatus.LOCKED,
-                    detail="Invitation was not found.")).response
```

Comment:
I don't know exactly the reason behind this, otherwise we can re-introduce this as `if acc_id >= max_account_id`.